assignments/Assignment5.js

```diff
@@ -0,0 +1,53 @@
+// Use nested loops to create diamond pattern
+
+function diamondPattern(centerWidth) {
+    //let centerWidth = 5;
+    let s = "";
+
+    // Build top of diamond
+    for (i=1; i<=centerWidth; i+=2) {
```

Comment:
nit: variables declared within for loops should be declared using the `let` keyword